grails-app/conf/Config.groovy

```diff
@@ -128,7 +128,7 @@ openId {
 }
 
 featureToggles {
-    // someFeature = <boolean>
+    filterSource = 'database' // Todo - DN: This can go back to being a boolean when we've removed loading filters from the DB
```

Comment:
This will become a boolean when the database option is removed (currently a string because there are 3 options). But as discussed I will move it out of the feature toggles block as it is not really a feature toggle.